sadcowpng/sharedspaces/static/spaces.css

```diff
@@ -0,0 +1,32 @@
+  body {
+      text-align: center;
+  }
+
+  .create_space_form{
+      display: inline-block;
+      width: 800px;
+  }
+
+  .create_space_table{
+      width: 800px;
+  }
+
+  .space_submit_button{
+      margin-top: 10px;
+      width: 400px;
+  }
+
+   .update_space_form{
+       display: inline-block;
+       width: 800px;
+   }
+
+  .update_space_table{
+      width: 800px;
+  }
+
+  .space_submit_button{
+      margin-top: 10px;
+      width: 400px;
+  }
+
```

Comment:
I might need to steal some of your stylings in the other pages and just make it match up with yours.